src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -1,13 +1,16 @@
 import _ from '../../../../platform/utilities/data';
 
 import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
+// NOTE: Easier to run schema locally with hot reload for dev
+// import fullSchema526EZ from '/User/Some/Path/vets-json-schema/dist/21-526EZ-schema.json';
 import fileUploadUI from '../../../common/schemaform/definitions/file';
-
-import initialData from '../../../../../test/disability-benefits/526EZ/schema/initialData';
-
+import dateRangeUI from '../../../common/schemaform/definitions/dateRange';
 import IntroductionPage from '../components/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
 
+// TODO: Load live user prefill data from network
+import initialData from '../../../../../test/disability-benefits/526EZ/schema/initialData';
```

Comment:
This comment is quite clear already, can we clarify it by adding something like that initialData will be commented out or removed from here for production, and that its presence here is for dev purposes only?